example_plugins/keyboard/Cargo.toml

```diff
@@ -9,4 +9,9 @@ crate-type = ["cdylib"]
 [dependencies]
 cimvr_common = { path = "../../common" }
 cimvr_engine_interface = { path = "../../engine_interface" }
+cimvr_derive_macros = { path = "../../engine_derive_macros" }
```

Comment:
Maybe we ought to re-export `cimvr_engine_interface` and `cimvr_derive_macros` in `cimvr_common`, so that plugin devs don't have to add so much to Cargo.toml...

Reply:
I agree with re exporting the engine interface but I don't agree with bundling derive macros with Cimvr common. If we do we should make it a feature like most other libraries.

Reply:
After discussions in DMs I'm now convinced that we should re-export the macros and engine interface.